bricksrc/deprecations.py

```diff
@@ -1,6 +1,11 @@
 from .namespaces import BRICK, RDFS, SKOS, A
 
 deprecations = {
+    BRICK.Water_Tank: {
+        "version": "1.3.0",
+        "mitigation_message": "'Water_Tank' is deprecated as a space. 'Water_Storage_Tank' serves as it's replacement and is being moved to the equipment class.",
+        "replace_with": BRICK.Water_Storage_Tank,
+    },
```

Comment:
Not deprecating yet. Instead, change the definition of Water Tank to emphasize it is a space and we might move/get rid of it later

Reply:
The current definition is, "A space used to hold water". Would you like me to update the definition to include that it may be deprecated in the future?

Reply:
I can do it! Thanks